wandb/sdk/data_types/object_3d.py

```diff
@@ -225,6 +211,14 @@ def from_file(
         data_or_path: Union["TextIO", str],
         file_type: Optional["FileFormat3D"] = None,
     ) -> "Object3D":
+        """
+        Initializes Object3D from a file or stream
+
+        Arguments:
+            data_or_path (Union["TextIO", str]): A path to a file or a `TextIO` stream.
+            file_type (str): Specifies the format of data in `data_or_path`. Required when `data_or_path` is a
+                stream, ignored when `data_or_path` is a path (the type will be taken from the file extension.)
```

Comment:
Specifies the data format passed to `data_or_path`. Required when `data_or_path` is a `TextIO` stream.  This parameter is ignored if a file path is provided. The type is taken from the file extension.